### DISCUSSION TITLE
documentation - add javascript examples to docs and tutorials.

### DISCUSSION THREAD

**Discussion by karl-cardenas-coding** [remix-run/remix#2234] [Proposals]
At the moment, most of the example code snippets are in TypeScript, but there are others that are in Javascript. Ideally, it would be nice to have a tab for the user to toggle between the language of choice.  I've added an example screenshot of what HashiCorp does with their docs. In the example, the user can toggle between HCL or JSON.  [Source](https://www.consul.io/docs/discovery/services)

This way, you can keep the existing code snippets but not be opinionated when informing the end-user. This would also help improve the documentation experience as the inconsistency between the usage of the two languages could be removed. 

![image](https://user-images.githubusercontent.com/29551334/156949925-7bc5ba70-3d0f-40e2-a057-385530d14d3c.png)

**Comment by kiliman** [comment]
I wonder if adding something like this to the docs will allow them to dynamically remove the TypeScript types from the examples. https://github.com/cyco130/detype
